notebooks/analysis/metpy-soundings-calculations.ipynb

```diff
@@ -0,0 +1,381 @@
+    "\n",
+    "As you can see in the docstring, the `cape_cin` function requires a `parcel_profile`, which is the temperature profile of a parcel.\n",
+    "No worries, as MetPy can compute this for us using the `parcel_profile` function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "prof = mpcalc.parcel_profile(p, T[0], Td[0]).to('degC')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can compute the CAPE and CIN values with one function call:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cape, cin = mpcalc.cape_cin(p, T, Td, prof)\n",
+    "print(\"CAPE: {:4.1f}\".format(cape))\n",
+    "print(\"CIN: {:4.1f}\".format(cin))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Visually represent calculated parameters <a name=\"plot\" />\n",
+    "\n",
+    "\n",
+    "The last step will be to visualize our computed parameters.\n",
+    "Building upon the [MetPy SkewT](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/visualization/MetPy-SkewT.ipynb) notebook, we can create a SkewT diagram using matplotlib.\n",
+    "See the comments within the code block below for an explanation about how each element is added to the diagram:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#####################\n",
+    "# Basic SkewT setup #\n",
+    "#####################\n",
+    "\n",
+    "fig = plt.figure(figsize=(9, 9))\n",
+    "add_metpy_logo(fig, 115, 100)\n",
+    "skew = SkewT(fig, rotation=45)\n",
+    "\n",
+    "# Plot the data using normal plotting functions, in this case using\n",
+    "# log scaling in Y, as dictated by the typical meteorological plot.\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "# compute u and v from windspeed and wind direction\n",
+    "u, v = mpcalc.wind_components(wind_speed, wind_dir)\n",
+    "skew.plot_barbs(p, u, v)\n",
+    "skew.ax.set_ylim(1000, 100)\n",
+    "skew.ax.set_xlim(-40, 60)\n",
+    "\n",
+    "# Add the relevant special lines\n",
+    "skew.plot_dry_adiabats()\n",
+    "skew.plot_moist_adiabats()\n",
+    "skew.plot_mixing_lines()\n",
+    "\n",
+    "#####################################\n",
+    "# Visualize the computed parameters #\n",
+    "#####################################\n",
+    "\n",
+    "# Plot the location of the LCL as black dot\n",
+    "skew.plot(lcl_pressure, lcl_temperature, 'ko', markerfacecolor='black')\n",
+    "\n",
+    "# Plot the profile parcel as black line\n",
+    "skew.plot(p, prof, 'k', linewidth=2)\n",
+    "\n",
+    "# Shade areas of CAPE and CIN\n",
+    "skew.shade_cin(p, T, Td)\n",
```

Comment:
Passed in `Td` instead of `prof` here, so you end up shading CIN between the parcel trace and the dewpoint trace instead of between parcel and temperature. You are able to provide `Td` after `prof` if you want `shade_cin` to exclude area below the LFC and above the EL, but it seems that's not totally right right now (it's excluding CIN between the LCL and the LFC for some reason for me locally? I'll look into this more later*), so I'd say probably best to just drop it in this example to just `skew.shade_cin(p, T, prof)`

Reply:
Done :-)